specification/resources/resource-manager/Microsoft.Authorization/stable/2019-01-01/policySetDefinitions.json

```diff
@@ -0,0 +1,625 @@
+{
```

Comment:
I dont see any delta from 2018-05-01 api version. Is that correct?

Reply:
Yes, other than the mode change and minor description changes, there are no major changes.